xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/editors/autosave.js

```diff
@@ -31,7 +31,7 @@ editors.AutoSave = Class.create({
     /** Is the autosave enabled ? */
     enabled: false,
     /** If enabled, how frequent are the savings */
-    frequency: 5, // minutes
+    interval: 5, // minutes
```

Comment:
This is a breaking API change as this is a public API, e.g., documented at https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/Autosave/. I would suggest not modifying the API and just modifying the displayed text.